src/components/hero.js

```diff
@@ -1,45 +1,49 @@
-import React from "react"
+import React, { Component } from "react"
 
-const Hero = () => (
-  <div className="cmp-hero">
-    <h1 className="cmp-hero__heading">
-      <a className="cmp-hero__logo" href="https://seesparkbox.com">
-        <img src="/images/logo.svg" alt="Sparkbox" />
-      </a>
-      <span className="cmp-hero__title">Design Systems</span>
-      <span className="cmp-hero__survey">Survey</span>
-      <span className="cmp-hero__year">2019 Edition</span>
-    </h1>
+class Hero extends Component {
```

Comment:
Wondering what the advantage was turning this into a class instead of leaving it as a function component? 🤔 From what I'm seeing in the [React docs](https://reactjs.org/docs/components-and-props.html), these two are equal. Not a reason to change it, just wondering what your thought process was 👍

Reply:
Ah, I see now. We probably need to add the mounting and intersection observer to this component to make `cmp-hero--animated` work, and it has to be a class to do that.

Reply:
Oops. I must have did some undos and saved and didn’t double check. I’ll get things cleaned up.